test/e2e-go/features/participation/participationExpiration_test.go

```diff
@@ -50,13 +50,11 @@ func testExpirationAccounts(t *testing.T, fixture *fixtures.RestClientFixture, f
 
 	transactionFee := minTxnFee
 	amountToSendInitial := 5 * minAcctBalance
+
 	fixture.SendMoneyAndWait(initialRound, amountToSendInitial, transactionFee, richAccount, sAccount, "")
-	amt, err := sClient.GetBalance(sAccount)
-	a.NoError(err)
 	sNodeStatus, err := sClient.Status()
 	a.NoError(err)
 	seededRound := sNodeStatus.LastRound
-	a.Equal(amountToSendInitial, amt)
```

Comment:
I think that this check need to be kept : it verifies that the above transaction truly takes place.